includes/class-sensei-admin.php

```diff
@@ -113,6 +114,22 @@ public function admin_menu() {
 		add_submenu_page( 'edit.php?post_type=lesson', __( 'Order Lessons', 'sensei-lms' ), __( 'Order Lessons', 'sensei-lms' ), 'edit_published_lessons', $this->lesson_order_page_slug, array( $this, 'lesson_order_screen' ) );
 	}
 
+	/**
+	 * Add Course order page to admin panel.
+	 *
+	 * @since  4.0.0
+	 */
+	public function add_course_order() {
+		add_submenu_page(
+			null, // Hide in menu.
```

Comment:
@donnapep [suggested](https://github.com/Automattic/sensei/pull/4646#discussion_r790913960) that we use `options.php` to hide menus. This makes things consistent, but I suspect you might have issues with changing the active menu in the admin nav.

Reply:
You're right: I can't highlight the menu item. And the Sensei LMS menu item is collapsed.
@donnapep I think our case differs significantly from Setup Wizard, especially in terms of menu, and (at the moment) I didn't find any pitfalls of providing null. I'll spend some more time investigating potential issues.

Reply:
If it works with `null`, no problem. We can continue using it.